src/onvif/Subscriber.js

```diff
@@ -35,8 +35,22 @@ export default class OnvifSubscriber {
 
     this.logger.info(`Successfully connected.`);
 
-    this.cam.on('event', camMessage => {
-      this.onEvent(this.name, camMessage);
-    });
+    this.subscribeOnCam();
   };
-}
\ No newline at end of file
+
+  subscribeOnCam = () => {
+    setTimeout(() => {
+      this.cam.createPullPointSubscription((err, _subscription, _xml) => {
+          if (!err) {
+            this.logger.info(`CreatePullPointSubscription ${this.name}`);
+            this.cam.on('event', camMessage => {
+              this.onEvent(this.name, camMessage);
+            });
+         } else {
+            this.logger.error(`CreatePullPointSubscription ${this.name}`);
+            this.subscribeOnCam();
+         }
+      });
+    }, Math.floor(Math.random() * Math.floor(5000)));
```

Comment:
Hi! Is there a reason for the random interval? Could we set it to `0` and expect node to run this next tick?

Reply:
Hi. This nice work, if a lot of cameras couldn't get pull-point. Work for me ) 
I can take it to config file.

Reply:
I see! The spread makes sense then. Just one small request to correct:
```
Math.floor(Math.random() * Math.floor(5000)));
```

to
```
Math.floor(Math.random() * 5000));
```

Reply:
Thank you!

Reply:
Sorry fo delay, fix. Happy New Year! )